### ISSUE TITLE
Intro and show packs shouldn't be separate entry points

### ISSUE THREAD

**Issue by rickmcgeer** [engageLively/ibn-battuta-new#2]
The flow goes straight from the intro to showing the packs, then into fez

**Comment by merryman** [comment]
So the part we we walk around and explore fez is gone now?

**Comment by rickmcgeer** [comment]
No, the part where we walk around fez is between showing the packs and buying one.  The idea is that the user picks up the information he needs to buy a pack as he walks around Fez
